lib/widgets/compose_box.dart

```diff
@@ -867,18 +867,16 @@ class _ComposeBoxContainer extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    ColorScheme colorScheme = Theme.of(context).colorScheme;
+    final designVariables = DesignVariables.of(context);
```

Comment:
I have been wondering about that too. I had 3 commits in an earlier revision, where 0e99b76f508a9759323c59f05a1477673a04938b implements the entire redesign, which would be the alternative to the now more granular commit sequence. I guess it might help to use that structure with a checklist of redesign details.

Or maybe we can try to strike a balance between organizing these changes in a single a big commit or multiple small ones. There are some arguably distinctive parts of the redesign like the icon changes that might be extracted.

Reply:
Happy to defer to @gnprice's judgment here. 🙂

Reply:
I haven't yet read this PR in detail, but I'm hopeful that this redesign can be done as a series of commits that are usefully broken up from each other, so that a reader is able to focus on one aspect at a time.

If there are commits where it doesn't seem possible to write a clean commit message, though, that'd be a sign that those commits should be squashed with related changes (if that enables a clean commit message).